artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/ActiveMQClientLogger.java

```diff
@@ -395,6 +395,11 @@
            format = Message.Format.MESSAGE_FORMAT)
    void unableToCheckEpollAvailability(@Cause Throwable e);
 
+   @LogMessage(level = Logger.Level.WARN)
+   @Message(id = 212080, value = "Unable to check IoUring availability ",
```

Comment:
Please put in correct order place in file based on id. See other comment for why